src/Installers/Windows/Wix.targets

```diff
@@ -23,7 +23,7 @@
     </GenerateGuid>
 
     <PropertyGroup>
-      <DefineConstants>$(DefineConstants);ProductCode=$(ProductCode);UpgradeCode=$(UpgradeCode)</DefineConstants>
+      <!-- <DefineConstants>$(DefineConstants);ProductCode=$(ProductCode);UpgradeCode=$(UpgradeCode)</DefineConstants> -->
```

Comment:
Please undo before merging. This is important for the other installers in this repo.